### ISSUE TITLE
Feature: Support HashMap<_, _, ahash::RandomState>

### ISSUE THREAD

**Issue by cauliyang** [Jij-Inc/pyo3-stub-gen#54]
I am wondering if we can support HashMap<_, _, hash::Random State> from https://docs.rs/ahash/latest/ahash/. pyo3 supports this type as well. BTW, thanks for your effort, and the library's really helpful!

```
error[E0277]: the trait bound `HashMap<i64, char, ahash::RandomState>: PyStubType` is 
not satisfied
   --> crates/deepbiop-fq/src/python.rs:457:15
    |
457 |     id_table: HashMap<i64, char>,
    |               ^^^^^^^^^^^^^^^^^^ the trait `PyStubType` is not implemented for `
HashMap<i64, char, ahash::RandomState>`
    |
    = help: the trait `PyStubType` is implemented for `HashMap<Key, Value>`

error[E0277]: the trait bound `HashMap<std::string::String, Predict, ahash::RandomStat
e>: PyStubType` is not satisfied
    |
459 | ) -> Result<HashMap<String, Predict>> {
    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `PyStubType` is not implemented 
for `HashMap<std::string::String, Predict, ahash::RandomState>`, which is required by 
`Result<HashMap<std::string::String, Predict, ahash::RandomState>, anyhow::Error>: PyS
tubType`
    |
    = help: the trait `PyStubType` is implemented for `HashMap<Key, Value>`
    = note: required for `Result<HashMap<std::string::String, Predict, ahash::RandomSt
ate>, anyhow::Error>` to implement `PyStubType`

For more information about this error, try `rustc --explain E0277`.

```

**Comment by termoshtt** [comment]
Added in #55 , and released as 0.5.2. Thanks feedback!

**Comment by cauliyang** [comment]
Thanks very much!
